### ISSUE TITLE
[suggestion] Add version to info page

### ISSUE THREAD

**Issue by akhepcat** [pimoroni/badger2040#88]
on the info screen, it sure would be helpful to have the current badgeros  version somewhere.

Maybe line 18?

display.text("badgerOS v0.0.5", 3, 4, WIDTH, 1)

That'll help remind us when we're a bit out of date and need to back up our modifications and refresh the OS

**Comment by rgsteele** [comment]
I agree! This has actually been suggested previously: https://github.com/pimoroni/badger2040/issues/15.

**Comment by Gadgetoid** [comment]
Added, albeit pending a release!
